src/Command/SniffCommand.php

```diff
@@ -86,6 +84,8 @@ private function getFormatter(string $type): FormatterInterface
         switch ($type) {
             case 'json':
                 return new JsonFormatter();
```

Comment:
Can't we pass `$input->getOption('pretty')` to the constructor? Instead of passing it to every `->format` call.

Reply:
fixed